test/parallel/test-http-information-processing.js

```diff
@@ -2,22 +2,21 @@
 require('../common');
 const assert = require('assert');
 const http = require('http');
-const Countdown = require('../common/countdown');
 
-const test_res_body = 'other stuff!\n';
-const countdown = new Countdown(3, () => server.close());
+const testResBody = 'other stuff!\n';
+const MESSAGE_COUNT = 2;
 
 const server = http.createServer((req, res) => {
-  console.error('Server sending informational message #1...');
-  res.writeProcessing();
-  console.error('Server sending informational message #2...');
-  res.writeProcessing();
+  for (let i = 0; i < MESSAGE_COUNT; i++) {
+    console.error(`Server sending informational message #${i}...`);
```

Comment:
If the `console.error()` is not part of the test, I'd prefer this to be replaced with a `debug()`... e.g.

```
const debug = require('util').debuglog('test');
// ...
debug(`Server sending informational message #${i}...`);
```

Reply:
One day, we'll have `console.debug()` working such that it doesn't output anything unless `NODE_DEBUG` is set or something like that.

Reply:
Replaced all console.error() with debug() statements

Reply:
hmm... still seeing the `console.error()` statements here...